tests/versioned/segments.tap.js

```diff
@@ -37,23 +55,14 @@ tap.test('Next.js', (t) => {
 
     const URI = '/ssr/people'
 
-    const res = await helpers.makeRequest(URI)
+    const res = await helpers.makeRequest(URI, app.server.port)
```

Comment:
I'm sure this is is harmless, but why is it necessary? Why are we passing in the port now in all requests in the tests?